pkgs/tools/graphics/lepton/default.nix

```
@@ -1,14 +1,14 @@
 { stdenv, fetchFromGitHub, cmake, git, glibc }:
 
 stdenv.mkDerivation rec {
-  version = "1.2.1";
+  version = "git-2019-08-20";
   pname = "lepton";
```

please set `pname` to `lepton-unstable` and simply use the date as `version`. See: https://github.com/NixOS/nixpkgs/issues/68518
```suggestion
  version = "2019-08-20";
  pname = "lepton-unstable";
```
besides that, LGTM

---

I'll fix that. I thought the git naming was normal as I saw it in a bunch of different packages but this makes more sense.